.github/workflows/build-deb-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Test Deb Package
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
+  pull_request:
+jobs:
+  build-test-deb-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - run: chmod 755 homeassistant-supervised/DEBIAN/p*
+      - run: dpkg-deb --build --root-owner-group homeassistant-supervised
+      - name: Upload
+        uses: actions/upload-artifact@v2
+        with:
+          name: homeassistant-supervised.deb
+          path: homeassistant-supervised.deb
+          retention-days: 15
```

Comment:
```suggestion
```

These files are small, and this repo does not have that high of a volume of builds, the default of 90 is fine